en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
     
 4. Upload the public certificate of Identity Server under **Identity Provider Public Certificate**.
 
-5. Configure **Federated authenticators** > **SAML2 Web SSO Configurations**
+5.  Click and expand the **Claim Configuration** section, and click and expand **Basic Claim Configuration**.
+    1. Tick `Define Custom Claim Dialect` radio box.
+    2. Click `Add Claim Mapping` and add the following data.
+        * Identity Provider Claim URI: `http://wso2.org/claims/groups`
+        * Local Claim URI: `http://wso2.org/claims/roles`
+    3. In the dropdown next to the **Role Claim URI** label, select `http://wso2.org/claims/groups`
+
+6.  Click and expand the **Role Configuration** section.
+    1. Click **Add role Mapping**.
+    2. Add the following row 
+        * Identity Provider Role: `Internal/creator`
```

Comment:
@Tharsanan1 , what do you mean by this line?

"Add the following row "

Reply:
After we click `Add role Mapping` we get a table. in the table we should give `Internal/creator` for Identity Provider Role column